opennms-base-assembly/src/main/filtered/etc/datacollection/snmp-informant-exchange.xml

```diff
@@ -332,103 +233,11 @@
-      <mibObj oid=".1.3.6.1.4.1.9600.1.1.3.1.14" instance="sinfNetInstance" alias="netPktsPerSec" type="Gauge32"/>
-   </group>
-   <!-- This group requires the pay-for SNMP Informant Advanced agent -->
-   <group name="snmpinformant-adv-network" ifType="all">
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.52.1.1" instance="ainfNetInstance" alias="ainfNetInstance" type="string"/>
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.52.1.2" instance="ainfNetInstance" alias="ainBytesRcvdPerSec" type="Gauge32"/>
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.52.1.3" instance="ainfNetInstance" alias="ainBytesSentPerSec" type="Gauge32"/>
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.52.1.4" instance="ainfNetInstance" alias="ainBytesTotalPerSec" type="Gauge32"/>
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.52.1.5" instance="ainfNetInstance" alias="ainCurrentBandwidth" type="Gauge32"/>
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.52.1.6" instance="ainfNetInstance" alias="ainOutputQueueLen" type="Gauge32"/>
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.52.1.7" instance="ainfNetInstance" alias="ainPktsOutDiscard" type="Counter"/>
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.52.1.8" instance="ainfNetInstance" alias="ainPktsOutErrors" type="Counter"/>
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.52.1.9" instance="ainfNetInstance" alias="ainPktsRcvdDiscard" type="Counter"/>
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.52.1.10" instance="ainfNetInstance" alias="ainPktsRcvdErrors" type="Counter"/>
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.52.1.11" instance="ainfNetInstance" alias="ainPktsRcvdNUPerSec" type="Gauge32"/>
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.52.1.12" instance="ainfNetInstance" alias="ainPktsRcvdUCPerSec" type="Gauge32"/>
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.52.1.13" instance="ainfNetInstance" alias="ainPktsRcvdUnknown" type="Counter32"/>
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.52.1.14" instance="ainfNetInstance" alias="ainPktsRcvdPerSec" type="Gauge32"/>
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.52.1.15" instance="ainfNetInstance" alias="ainPktsSentNUPerSec" type="Gauge32"/>
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.52.1.16" instance="ainfNetInstance" alias="ainPktsSentUCPerSec" type="Gauge32"/>
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.52.1.17" instance="ainfNetInstance" alias="ainPktsSentPerSec" type="Gauge32"/>
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.52.1.18" instance="ainfNetInstance" alias="ainPktsPerSec" type="Gauge32"/>
-   </group>
-   <!-- This group requires the pay-for SNMP Informant Advanced agent -->
-   <group name="snmpinformant-adv-termsvc-scalars" ifType="ignore">
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.81.1" instance="0" alias="sinfTsActiveSess" type="gauge"/>
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.81.2" instance="0" alias="sinfTsInactiveSess" type="gauge"/>
-      <mibObj oid=".1.3.6.1.4.1.9600.1.2.81.3" instance="0" alias="sinfTsTotalSess" type="gauge"/>
-   </group>
+   <systemDef name="Microsoft Windows SNMP Informant">
```

Comment:
I would rename the `systemDef` to be `Microsoft Windows SNMP Informant Exchange`. Remember that it is mandatory that all `systemDefs` have unique names.